### PR TITLE
tweaks to dockerfile

### PR DESCRIPTION
**Pull request by chrislovecnm**
Do we only need kubectl and helm on this?  What are we doing with gcloud on this docker?

### CODE REVIEW

**Comment by viglesiasce** [comment]
We don't need them today but we will tomorrow when we added functional testing. I'd prefer to keep them in for now.

**Comment by chrislovecnm** [comment]
I tested the docker locally and bumped the version number of the docker in the publish script.  Let me know if this is overkill, but I love me some repeatable processes 😀

**Comment by chrislovecnm** [comment]
@viglesiasce so I broke the test, by bumping the version on the test script.  Insert rimshot here.  Causality Delima ...  

Options? Let me back out the version bump.

**Comment by chrislovecnm** [comment]
So I am seeing Jenkins do a build in GitHub and then the plugin section disappears out of the PR page.  Officially confused!  Oh well ... lol.  Let me know what needs to be tweaked.

File: test/publish-docker.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```
Comment (viglesiasce) [inline test/publish-docker.sh:1]: I would rename this file publish-test-image.sh

**Comment by viglesiasce** [comment]
@chrislovecnm to see the results of tests (that passed). Click the "Show all checks" button,

**Comment by chrislovecnm** [comment]
Ahhhhhh lol. I will do a rebase. And change the file name!

**Comment by chrislovecnm** [comment]
@viglesiasce if you want to publish the docker, and if I can access it, I can bump the version in the `e2e.sh` script.

File: test/e2e.sh [hunk 2]
```diff
@@ -13,5 +13,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-docker run -v `pwd`:/src gcr.io/kubernetes-charts-ci/test-image:v1.5 /src/test/changed.sh
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# TODO should we inject this.  This is creating problems bumping the Docker version
+IMAGE_VERSION="test-images:v1.5"
```
Comment (viglesiasce) [inline test/e2e.sh]: this needs to be test-image

**Comment by viglesiasce** [comment]
LGTM Thanks @chrislovecnm

**Pull request merged**